src/QuestionViewer/index.js

```diff
@@ -21,10 +21,14 @@ class QuestionViewer extends Component {
       <div>
         {loading && <div>Loading...</div>}
         <h1>Q{question.qno}: {question.title}</h1>
+        {error && <div className="error">ERROR: {error}</div>}
         <p>
           {question.body}
         </p>
-        {error && <div className="error">ERROR: {error}</div>}
+        <ul class="Nav">
+        <li><a rel="prev" href="question/1" id="prv">&lt; Prev</a></li>
```

Comment:
What I wanted is that you use `props.params.qno` and find the previous and the next question number. Note that this would be a string so `parseInt` it at the start of the render function and use that with [template strings](https://github.com/lukehoban/es6features#template-strings) in the url routes.